plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -20,6 +20,31 @@
         <exclusionFilter pattern="com/google/apphosting/**" />
       </libraryFile>
     </library>
+    <library
+          id="appengine-endpoints"
+          name="App Engine Endpoints"
```

Comment:
Is this a UI message to be localized? The same goes with the `appengine-api` and `objectify` libraries.